octopoes/octopoes/models/ooi/config.py

```diff
@@ -0,0 +1,21 @@
+from typing import Literal
+
+from octopoes.models import OOI, Reference
+from octopoes.models.persistence import ReferenceField
+
+
+class Config(OOI):
+    object_type: Literal["Config"] = "Config"
+
+    ooi: Reference = ReferenceField(OOI)
+    config: str
+
+    @property
+    def natural_key(self) -> str:
+        return str(self.ooi)
```

Comment:
Can't OOIs have multiple configs applying to different bits?

Reply:
Hmm could make sense indeed... maybe best to add bit_id as attr str

Reply:
Thought this would save a query to get the config by just adding "Config|" to primary key of OOI. But loses flexibility indeed

Reply:
So, current setup is OK?